### PR TITLE
Nodejs week1 homework

### PR DESCRIPTION
**Pull request by mag-da-lenka**

### CODE REVIEW

File: nodejs/week1/mealsharing_app_1/src/backend/routes/cheap-meals.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+// data source
+const jsonMeals = require(__dirname + '/../data/meals.json');
+const jsonReviews = require(__dirname + '/../data/reviews.json');
+
+
+const cheapMealsFiltered = jsonMeals.filter((aCheapMeal) => aCheapMeal.price < 107);
+
+cheapMealsFiltered.forEach((aMeal) => {
+
+    aMeal.review = [];
```
Comment (Hatko) [inline nodejs/week1/mealsharing_app_1/src/backend/routes/cheap-meals.js]: You could write it in a simpler way. Check this comment for a reference:

https://github.com/krithi-kondana/hyf-homework/pull/30#discussion_r382179162

The same comment applies to other files as well 🙂

Comment (mag-da-lenka) [reply nodejs/week1/mealsharing_app_1/src/backend/routes/cheap-meals.js]: You can write it shorter with filter:
`meal.review = reviewsJson.filter(mealReview => mealReview.mealId === meal.id)[0]`
If you have questions regarding how it works, ask me on Sunday ;)
The same code applicable to all the places you used similar logic. 
---> copy-paste ---> to be implemented :) Thank you!

File: nodejs/week1/circle_calculator/package.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "circle_calculator",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
[the remark this answers is not in the document]
Comment (mag-da-lenka) [reply nodejs/week1/circle_calculator/package.json]: Ohhh, of course... :D

**Comment by mag-da-lenka** [comment]
Thank you for your time and evaluation. Changes will be implemented soon :)

**Comment by mag-da-lenka** [comment]
I have implemented the feedback. Waiting for the green light to merge with master. Thank you 👍

**Pull request merged**